### ISSUE TITLE
screen touch freeze

### ISSUE THREAD

**Issue by paulobreim** [flutter/flutter#69878]
## Steps to Reproduce

1. Run `flutter create bug`.
2. Update the files as follows: ... 
3. ... 

**Expected results:** 

**Actual results:** 

<details>
  <summary>Logs</summary>

```
```

```
```

```
```

</details>

**Comment by paulobreim** [comment]
When using on android 10 (Samsung S20), the call returns normally but the cell phone screen freezes and only resolves with reset.
using flutter_webview_plugin work well.

**Comment by darshankawar** [comment]
@paulobreim 
Please provide `flutter doctor -v` and a minimal complete reproducible code sample that shows the issue.
Thanks.

**Comment by paulobreim** [comment]
[pcb@pcb-asus ~]$ flutter doctor -v
[✓] Flutter (Channel stable, 1.22.3, on Linux, locale pt_BR.UTF-8)
    • Flutter version 1.22.3 at /home/pcb/development/flutter
    • Framework revision 8874f21e79 (8 days ago), 2020-10-29 14:14:35 -0700
    • Engine revision a1440ca392
    • Dart version 2.10.3

[✓] Android toolchain - develop for Android devices (Android SDK version 30.0.1)
    • Android SDK at /home/pcb/Android/Sdk
    • Platform android-30, build-tools 30.0.1
    • Java binary at: /opt/android-studio/jre/bin/java
    • Java version OpenJDK Runtime Environment (build 1.8.0_242-release-1644-b3-6222593)
    • All Android licenses accepted.

[✓] Android Studio (version 4.0)
    • Android Studio at /opt/android-studio
    • Flutter plugin version 49.0.2
    • Dart plugin version 193.7547
    • Java version OpenJDK Runtime Environment (build 1.8.0_242-release-1644-b3-6222593)

[!] Connected device
    ! No devices available

--------------------------------------------
import 'dart:convert';
import 'package:flutter/material.dart';
import 'package:imoveissp/dados.dart';
import 'package:http/http.dart' as http;
import 'package:webview_flutter/webview_flutter.dart';
import 'dart:async';

pegaValorIPTU() async {
  var response;

  try {
    response = await http.get(accessurl).timeout(Duration(seconds: 15));
  } on TimeoutException {
    print('Timeout');
  } on Error catch (e) {
    print('Error: $e');
    respValoriIPTU = 'Sistema indisponível no momento.';
    return;
  }
  var aux = response.statusCode.toString();
  if (aux != "200") {
    respValoriIPTU = 'Sistema da PMSP indisponível no momento.';
  } else {
    respValoriIPTU = response.body;
  }
}

class Pagina4 extends StatefulWidget {
  @override
  Pagina4State createState() {
    return Pagina4State();
  }
}

class Pagina4State extends State<Pagina4> {
  WebViewController _webViewController;
  bool showLoading = true;

  void updateLoading(bool ls) {
    this.setState(() {
      showLoading = ls;
    });
  }

  @override
  Widget build(BuildContext context) {
    var scaffold = Scaffold(
      appBar: AppBar(
          title: Text('Informações Relativas ao IPTU'),
          backgroundColor: Color.fromRGBO(0x51, 0x7d, 0xa2, 1)),
      body: Stack(
        children: <Widget>[
          Container(
            child: WebView(
              javascriptMode: JavascriptMode.unrestricted,
              initialUrl: Uri.dataFromString(respValoriIPTU,
                      mimeType: 'text/html',
                      encoding: Encoding.getByName('utf-8'))
                  .toString(),
              onPageFinished: (data) {
                updateLoading(false);
              },
              onWebViewCreated: (webViewController) {
                _webViewController = webViewController;
                _webViewController.goForward();
              },
            ),
          ),
          (showLoading)
              ? Center(
                  child: CircularProgressIndicator(),
                )
              : Center()
        ],
      ),
    );
    return scaffold;
  }
}
-------------------------
Look the video attached.
The app receive the answer, show in the screen but after this freeze the touch.

[WhatsApp Video 2020-11-06 at 09.56.27.zip](https://github.com/flutter/flutter/files/5501417/WhatsApp.Video.2020-11-06.at.09.56.27.zip)

**Comment by darshankawar** [comment]
@paulobreim 
If I understand correctly, you are opening a http link using webview. When you tap on that link and come back to the webview, the screen freezes. Is it a correct interpretation of the issue ?

**Comment by paulobreim** [comment]
The step by step is:
1 - I call a url with using package:http/http.dart'. 
2 - The server mount a answer returning the data in the variable respValoriIPTU .
3 - I call the webview passing respValoriIPTU.
4 - This show correct in the user screen
5 - Afer this everything in the mobile freeze, need to do a reset.

See the attached video that I sent.

In my opinion this is a Android bug because an app could not freeze all mobile, the mobile is not rooted.
The app is in playstore for 1 year and this is the unique case. The user has a Samsung Galaxy S20 with Android 10. 

I did another test using flutter_webview_plugin and work without this problem.

**Comment by darshankawar** [comment]
@paulobreim 
Tried your code sample but getting below error while running the app:

```
I/flutter ( 6024): The following NoSuchMethodError was thrown building Pagina4(dirty, state: Pagina4State#a6b6b):
I/flutter ( 6024): The getter 'length' was called on null.
I/flutter ( 6024): Receiver: null
I/flutter ( 6024): Tried calling: length

```

<details>
<summary>code sample</summary>

```

import 'dart:convert';
import 'package:flutter/material.dart';
//import 'package:imoveissp/dados.dart';
import 'package:http/http.dart' as http;
import 'package:webview_flutter/webview_flutter.dart';
import 'dart:async';

pegaValorIPTU() async {
  var response;
  var respValoriIPTU;
  try {
    response = await http.get('https://flutter.dev').timeout(Duration(seconds: 15));
  } on TimeoutException {
    print('Timeout');
  } on Error catch (e) {
    print('Error: $e');
    respValoriIPTU = 'Sistema indisponível no momento.';
    return;
  }
  var aux = response.statusCode.toString();
  if (aux != "200") {
    respValoriIPTU = 'Sistema da PMSP indisponível no momento.';
  } else {
    respValoriIPTU = response.body;
  }
}

void main() {
  runApp(MyApp());
}

class MyApp extends StatelessWidget {
  @override
  Widget build(BuildContext context) {
    return MaterialApp(
      home: Pagina4(),
    );
  }

}




class Pagina4 extends StatefulWidget {
  @override
  Pagina4State createState() {
    return Pagina4State();
  }
}

class Pagina4State extends State<Pagina4> {
  WebViewController _webViewController;
  bool showLoading = true;

  String respValoriIPTU;

  void updateLoading(bool ls) {
    this.setState(() {
      showLoading = ls;
    });
  }

  @override
  Widget build(BuildContext context) {
    var scaffold = Scaffold(
      appBar: AppBar(
          title: Text('Informações Relativas ao IPTU'),
          backgroundColor: Color.fromRGBO(0x51, 0x7d, 0xa2, 1)),
      body: Stack(
        children: <Widget>[
          Container(
            child: WebView(
              javascriptMode: JavascriptMode.unrestricted,
              initialUrl: Uri.dataFromString(respValoriIPTU,
                  mimeType: 'text/html',
                  encoding: Encoding.getByName('utf-8'))
                  .toString(),
              onPageFinished: (data) {
                updateLoading(false);
              },
              onWebViewCreated: (webViewController) {
                _webViewController = webViewController;
                _webViewController.goForward();
              },
            ),
          ),
          (showLoading)
              ? Center(
            child: CircularProgressIndicator(),
          )
              : Center()
        ],
      ),
    );
    return scaffold;
  }
}

```

</details>

**Comment by pedromassangocode** [comment]
Without additional information, we are unfortunately not sure how to resolve this issue. 
We are therefore reluctantly going to close this bug for now. 
Please don't hesitate to comment on the bug if you have any more information for us; we will reopen it right away!  
Thanks for your contribution.

Could everyone who still has this problem please file a new issue with the exact description of what happens, logs, and the output of `flutter doctor -v`.
All system setups can be slightly different, so it's always better to open new issues and reference related issues.

**Comment by papmodern** [comment]
I experienced the exact problem on Galaxy S20 FE with Android 10. During development android touch freezes completely. I could not find any reproduction procedure yet.

**Comment by dgurudot** [comment]
We even tried the WAKE_LOCK but that didn't help on Samsung devices.

We have to press multiple times for any touch to be recognized on device although we could clearly see logs on console when we do press.

We added the following on all three AndroidManifest.xml files (debug, main, profile)

We also tried using the wakelock package and added the

await Wakelock.enable();

and it didn't help to resolve this issue.
